docs/tidb_http_api.md

```diff
@@ -582,7 +582,7 @@ timezone.*
 27. Get count {number} TiDB DDL job history information, start with job {id}
 
      ```shell
-     curl http://{TIDBIP}:10080/ddl/history?start_job_id={id}&limit={number}
+     curl "http://{TIDBIP}:10080/ddl/history?start_job_id={id}&limit={number}"
```

Comment:
There are a lot of uses of "&" in this file so we'd better unify.

Reply:
@hawkingrei could you help take a look again?

Reply:
Other urls can be changed together? It's not very uniform